e2e_testing/torchscript/xfail_sets.py

```diff
@@ -21,6 +21,21 @@
 }
 REFBACKEND_XFAIL_SET = COMMON_TORCH_MLIR_LOWERING_XFAILS
 
+EAGERMODEBACKED_XFAIL_SET = REFBACKEND_XFAIL_SET.union({
```

Comment:
as mentioned on chat, we should have clean fallback, so that eager mode "always Just Works"

Reply:
One problem here is the first block of tests that fail. In fact they do not fail (the outputs compare as equal but because [reporting.py](https://github.com/llvm/torch-mlir/blob/5d9222383ce80567911794e4104f336279911a87/python/torch_mlir_e2e_test/torchscript/reporting.py#L156-L159) checks dtypes you get inconsistent reporting and an exception [here](https://github.com/llvm/torch-mlir/blob/5d9222383ce80567911794e4104f336279911a87/python/torch_mlir_e2e_test/torchscript/reporting.py#L250)). As the comment says: this is because, for `.Scalar` variants, what actually happens is that a zero dim tensor `float64` is funneled into these ops at the `torch_dispatch` level.

Reply:
as discussed this is unfortunately an upstream issue; more documentation here https://github.com/pytorch/pytorch/issues/74400

Reply:
nit: Call this EAGER_MODE_XFAIL_SET to make it consistent with the `config_choices`.